onnx/reference/reference_evaluator.py

```diff
@@ -292,7 +292,7 @@ def _init(self) -> None:
         self.rt_inits_ = {}
         self.rt_nodes_ = []
         for init in self.inits_:
-            self.rt_inits_[init.name] = numpy_helper.to_array(init)
+            self.rt_inits_[init.name] = numpy_helper.to_array(init)  # type: ignore[union-attr,arg-type]
```

Comment:
I thought the new lintrunner check will be basically same as previous settings, but I do see there are some new errors -- May I understand what are the new checks that bring new error?

Reply:
It could be how the linter is run, but mypy was not known for being super deterministic from experience. It could be that in this run mypy has got more info about these vars that it can confidently emit an error vs previously treating the type as Any. Just a hypothesis. Most new errors seem legit to me.